### DISCUSSION TITLE
大佬按了开机自动启动，这个程序没反应怎么办

### DISCUSSION THREAD

**Discussion by huashenzjy** [skydevil88/XboxDownload#110] [General]
![PixPin_2024-12-01_15-08-26](https://github.com/user-attachments/assets/30d63e96-d24b-48f1-a4b0-df639c3640c1)
按了没反应

**Comment by skydevil88** [comment]
勾上保存就行，下次开机就会自动启动。

**Reply by huashenzjy** [reply]
> 勾上保存就行，下次开机就会自动启动。

大佬我试了很多次，勾上保存，重启，然后就没有启动，在任务管理器也没找到，我自己添加了在开机启动项，然后也没有启动

**Comment by skydevil88** [comment]
打开任务计划程序，检查上次运行结果。

**Comment by huashenzjy** [comment]
![image](https://github.com/user-attachments/assets/aed23a76-2479-4a7a-82fa-89e47820b511)
大佬你好，我在任务计划程序中没有找到它创建的项目，然后我现在自己创建了，发现现在打开程序，xboxdownload已经在开机启动打上勾，但还是需要手动开启监听

**Comment by skydevil88** [comment]
增加一个参数 Startup，还有常规里勾上使用最高权限运行。
<img width="479" alt="image" src="https://github.com/user-attachments/assets/f765ebec-efdc-4cc1-a23a-cfa438420495">

**Comment by huashenzjy** [comment]
感谢大佬的耐心回复，已经可以了，谢谢您
